include/Dialect/Secret/IR/SecretDialect.td

```diff
@@ -15,6 +15,13 @@ def Secret_Dialect : Dialect {
     custom types for arithmetic on secret integers of various bit widths.
   }];
 
+  let extraClassDeclaration = [{
+    /// Name of the attribute indicate whether an argument of a function is a
+    //secret.
+    constexpr const static ::llvm::StringLiteral
+        kArgSecretAttrName = "secret.secret";
```

Comment:
nit: any reason to call it `secret.secret` as opposed to just `secret`?

Reply:
Good question! It's because function arg attributes must be part of a dialect namespace, and so `secret` is interpreted as a 'builtin' / non-dialect attribute. I tried that first :)

```
heir/bazel-out/k8-dbg/bin/tests/secretize/multiple.mlir.test.runfiles/heir/tests/secretize/multiple.mlir:11:5: error: 'func.func' op arguments may only have dialect attributes
# |     func.func @main(%a: i1, %b: i1) -> () {
# |     ^
# | /usr/local/google/home/asraa/.cache/bazel/_bazel_asraa/dd6a35b586d0a2d65a5a8939e7a649cf/sandbox/linux-sandbox/5670/execroot/heir/bazel-out/k8-dbg/bin/tests/secretize/multiple.mlir.test.runfiles/heir/tests/secretize/multiple.mlir:11:5: note: see current operation: 
# | "func.func"() <{arg_attrs = [{secret}, {secret}], function_type = (i1, i1) -> (), sym_name = "main"}> ({
# | ^bb0(%arg0: i1, %arg1: i1):
# |   "func.call"(%arg0, %arg1) <{callee = @inner}> : (i1, i1) -> ()
# |   "func.return"() : () -> ()
# | }) : () -> ()
# `-----------------
```